### PR TITLE
fix: throw correct exception when project ID already exists

### PR DESCRIPTION
**Pull request by christyjacob4**
## What does this PR do?

Throw the correct exception when creating a project with a requested ID already exists 

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [ ] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/config/errors.php [hunk 1]
```diff
@@ -489,6 +489,11 @@
         'description' => 'Project with the requested ID could not be found. Please check the value of the X-Appwrite-Project header to ensure the correct project ID is being used.',
         'code' => 404,
     ],
+    Exception::PROJECT_ALREADY_EXISTS => [
+        'name' => Exception::PROJECT_ALREADY_EXISTS,
+        'description' => 'Project with the requested ID already exists.',
+        'code' => 403,
```
Comment (stnguyen90) [inline app/config/errors.php:495]: We use 409 for others so maybe use the same for this.

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

File: app/config/errors.php (hunk 1 above)
Comment (christyjacob4) [reply app/config/errors.php:495]: Yes true but @TorstenDittmann suggested using 403. Which I feel is also relevant in this case 🤔

Comment (TorstenDittmann) [reply app/config/errors.php:495]: should be 409, I might have remembered the wrong one when I mentioned it :)

File: tests/e2e/Services/Projects/ProjectsConsoleClientTest.php [hunk 2]
```diff
@@ -98,7 +99,37 @@ public function testCreateProject(): array
...
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), [
+            'projectId' => $projectId,
+            'name' => 'Project Duplicate',
+            'teamId' => $teamId,
+            'region' => 'default'
+        ]);
+
+        $this->assertEquals(403, $response['headers']['status-code']);
+        $this->assertEquals(403, $response['body']['code']);
```
Comment (christyjacob4) [inline tests/e2e/Services/Projects/ProjectsConsoleClientTest.php:130]:
```suggestion
        $this->assertEquals(409, $response['headers']['status-code']);
        $this->assertEquals(409, $response['body']['code']);
```

File: phpunit.xml [hunk 3]
```diff
@@ -6,7 +6,7 @@
          convertNoticesToExceptions="true"
          convertWarningsToExceptions="true"
          processIsolation="false"
-         stopOnFailure="false"
+         stopOnFailure="true"
```
Comment (christyjacob4) [inline phpunit.xml:9]:
```suggestion
         stopOnFailure="false"
```

File: app/config/errors.php (hunk 1 above)
Comment (christyjacob4) [inline app/config/errors.php:495]:
```suggestion
        'code' => 409,
```

Comment (christyjacob4) [reply app/config/errors.php]: Okay 👍  updated it

**Review by stnguyen90 — APPROVED** [review]
LGTM!

This is what the error looks like now:

image

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**